src/Aspire.Hosting/ApplicationModel/CustomResourceSnapshot.cs

```diff
@@ -68,6 +68,12 @@ public ResourceStateSnapshot? State
     /// </remarks>
     public HealthStatus? HealthStatus { get; private set; }
 
+    /// <summary>
+    /// Whether the health reports have been initialized in <see cref="ResourceNotificationService"/> to contain
+    /// the health check annotations for this resource.
+    /// </summary>
+    internal bool HealthAnnotationsInitialized { get; init; }
```

Comment:
Why do  you need this if it happens once?

Reply:
It's not needed, the usages were removed but I missed the actual property declaration. It's been removed